ASCIIEngine/Components/Component.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ASCIIEngine.Core.Components
+{
+    public class Component
+    {
+        internal virtual void Update(){}
```

Comment:
Component lacks Start() method

Reply:
It's not necessary in Start() method this case, so we can add Start() method later